Dockerfile

```diff
@@ -449,30 +449,6 @@ FROM scratch AS microsoft-db-keys
 COPY --from=microsoft-secureboot-database /DB/Certificates/MicCor*.der /db/
```

Comment:
we should be somwhere removing SBOMs from the rootfs, and then adding back merged ones? I don't see it?

Reply:
yes,  but SBOMs are already in say `pkg-kernel`, so does the cleanup script remove it?

Reply:
ok, so I'll take a look at the final SBOMs or even try to present them as resources to see how it looks

Reply:
It might be great though to create some separation to indicate talos.spdx.json and later extensions' SBOMs are the primary ones, and other ones are sort of sub-SBOMs

Reply:
Because you have all necessary info about Talos in its SBOM, and e.g. containerd's one gives you what containerd was built with. Or maybe we should drop them from the image altogether since that information is of not much use and can still be followed to by reading sources

Reply:
I mean there should be one for core Talos and others for extensions